cmd/mattermost/commands/roles_test.go

```diff
@@ -17,7 +17,7 @@ func TestAssignRole(t *testing.T) {
 	CheckCommand(t, "roles", "system_admin", th.BasicUser.Email)
 
 	if result := <-th.App.Srv.Store.User().GetByEmail(th.BasicUser.Email); result.Err != nil {
-		t.Fatal()
+		t.Fatal(result.Err)
```

Comment:
Out of the purpose of this PR, but as far as a require is a early termination, we could do this like:
```go
result := <-th.App.Srv.Store.User().GetByEmail(th.BasicUser.Email)
require.Nil(t, result.Err)
```